### ISSUE TITLE
aux_clm "clm-basic" tests fail as of ctsm5.3.0 because they need c13 in finidat

### ISSUE THREAD

**Issue by slevis-lmwg** [ESCOMP/CTSM#2787]
### Brief summary of bug

During ctsm5.3.0 testing, three aux_clm tests failed in the RUN phase because they were missing c13 in the finidat:
```
SMS_Lm1.f10_f10_mg37.I1850Clm50BgcCropCmip6waccm.derecho_gnu.clm-basic
ERS_Ly3.f10_f10_mg37.I1850Clm50BgcCropCmip6.derecho_intel.clm-basic
SMS_Ld2_D_PS.f09_g17.I1850Clm50BgcCropCmip6.derecho_intel.clm-basic_interp
```

### General bug information

**CTSM version you are using:** ctsm5.3.0

**Does this bug cause significantly incorrect results in the model's science?** No

**Configurations affected:** /basic and /basic_interp testmods directories

### Details of bug

Original post reporting the problem:
https://github.com/ESCOMP/CTSM/pull/2500#issuecomment-2372149377

**Comment by ekluzek** [comment]
Another way to handle this would be to update these tests to clm6_0. And I think that's actually what we should do to use the new clm6_0 finidat files as well as move more testing from Clm50 to Clm60.

**Comment by slevis-lmwg** [comment]
I had to update
```diff --git a/cime_config/config_compsets.xml b/cime_config/config_compsets.xml
index 538329e52..7618c51c9 100644
--- a/cime_config/config_compsets.xml
+++ b/cime_config/config_compsets.xml
@@ -197,15 +197,15 @@
   <!-- Primarily for testing the CMIP6DECK compset option -->
   <!-- Use the CMIP6 version (clm5_0) -->
   <compset>
-    <alias>I1850Clm50BgcCropCmip6</alias>
-    <lname>1850_DATM%GSWP3v1_CLM50%BGC-CROP-CMIP6DECK_SICE_SOCN_MOSART_SGLC_SWAV</lname>
+    <alias>I1850Clm60BgcCropCmip6</alias>
+    <lname>1850_DATM%GSWP3v1_CLM60%BGC-CROP-CMIP6DECK_SICE_SOCN_MOSART_SGLC_SWAV</lname>
   </compset>
 
   <!-- Primarily for testing the CMIP6WACCMDECK compset option -->
   <!-- Use the CMIP6 version (clm5_0) -->
   <compset>
-    <alias>I1850Clm50BgcCropCmip6waccm</alias>
-    <lname>1850_DATM%GSWP3v1_CLM50%BGC-CROP-CMIP6WACCMDECK_SICE_SOCN_MOSART_SGLC_SWAV</lname>
+    <alias>I1850Clm60BgcCropCmip6waccm</alias>
+    <lname>1850_DATM%GSWP3v1_CLM60%BGC-CROP-CMIP6WACCMDECK_SICE_SOCN_MOSART_SGLC_SWAV</lname>
```
Now troubleshooting new error:
`ERROR: No description found for comp_class lnd matching compsetname 1850_DATM%GSWP3v1_CLM60%BGC-CROP-CMIP6DECK_SICE_SOCN_MOSART_SGLC_SWAV_SESP in file /glade/derecho/scratch/slevis/temp_work/new_rawdata/cime_config/config_component.xml, expected match in ['CLM60'] % ['SP', 'SP-NOANTHRO', 'BGC-NOANTHRO', 'BGC', 'BGC-CROP', 'FATES', 'FATES-SP', 'NWP-SP', 'NWP-BGC-CROP']`

**Comment by slevis-lmwg** [comment]
I got past the last error with this update
```diff
--- a/cime_config/config_component.xml
+++ b/cime_config/config_component.xml
@@ -14,7 +14,7 @@
   <!-- Descriptions of all the different valid configurations for different model versions -->
   <description modifier_mode="1">
     <desc lnd="CLM45[%SP][%SP-VIC][%BGC][%BGC-CROP][%FATES][%BGCDV][%BGCDV-CROP]"                  >clm4.5:</desc>
-    <desc lnd="CLM50[%SP][%SP-VIC][%SP-NOANTHRO][%BGC-NOANTHRO][%BGC][%BGC-CROP][%FATES][%BGCDV][%BGCDV-CROP][%BGC-CROP-CMIP6DECK][%BGC-CROP-CMIP6WACCMDECK][%NWP-SP][%NWP-BGC-CROP]">clm5.0:</desc>
+    <desc lnd="CLM60[%SP][%SP-VIC][%SP-NOANTHRO][%BGC-NOANTHRO][%BGC][%BGC-CROP][%FATES][%BGCDV][%BGCDV-CROP][%BGC-CROP-CMIP6DECK][%BGC-CROP-CMIP6WACCMDECK][%NWP-SP][%NWP-BGC-CROP]">clm6.0:</desc>
     <desc lnd="CLM51[%SP][%SP-NOANTHRO][%BGC-NOANTHRO][%BGC][%BGC-CROP][%FATES][%FATES-SP][%NWP-SP][%NWP-BGC-CROP]">clm5.1:</desc>
     <desc lnd="CLM60[%SP][%SP-NOANTHRO][%BGC-NOANTHRO][%BGC][%BGC-CROP][%FATES][%FATES-SP][%NWP-SP][%NWP-BGC-CROP]">clm6.0:</desc>
     <desc option="SP"              >Satellite phenology:</desc>
@@ -367,10 +367,10 @@
     <values match="last">
       <value                  compset="_CLM60%[^_]*FATES-SP[%_]"      >$COMP_ROOT_DIR_LND/cime_config/usermods_dirs/fates_sp</value>
       <value                  compset="_CLM51%[^_]*FATES-SP[%_]"      >$COMP_ROOT_DIR_LND/cime_config/usermods_dirs/fates_sp</value>
-      <value                  compset="_CLM50%[^_]*CMIP6DECK[%_]"     >$COMP_ROOT_DIR_LND/cime_config/usermods_dirs/cmip6_deck</value>
-      <value grid="l%1.9x2.5" compset="_CLM50%[^_]*CMIP6DECK[%_]"     >$COMP_ROOT_DIR_LND/cime_config/usermods_dirs/cmip6_nociso_deck</value>
-      <value                  compset="_CLM50%[^_]*CMIP6WACCMDECK[%_]">$COMP_ROOT_DIR_LND/cime_config/usermods_dirs/cmip6_waccm_deck</value>
-      <value grid="l%1.9x2.5" compset="_CLM50%[^_]*CMIP6WACCMDECK[%_]">$COMP_ROOT_DIR_LND/cime_config/usermods_dirs/cmip6_waccm_nociso_deck</value>
+      <value                  compset="_CLM60%[^_]*CMIP6DECK[%_]"     >$COMP_ROOT_DIR_LND/cime_config/usermods_dirs/cmip6_deck</value>
+      <value grid="l%1.9x2.5" compset="_CLM60%[^_]*CMIP6DECK[%_]"     >$COMP_ROOT_DIR_LND/cime_config/usermods_dirs/cmip6_nociso_deck</value>
+      <value                  compset="_CLM60%[^_]*CMIP6WACCMDECK[%_]">$COMP_ROOT_DIR_LND/cime_config/usermods_dirs/cmip6_waccm_deck</value>
+      <value grid="l%1.9x2.5" compset="_CLM60%[^_]*CMIP6WACCMDECK[%_]">$COMP_ROOT_DIR_LND/cime_config/usermods_dirs/cmip6_waccm_nociso_deck</value>
```

**Comment by slevis-lmwg** [comment]
Next step: Open PR

**Comment by ekluzek** [comment]
@wwieder @slevis-lmwg and I discussed this one. And we realized that this is pointing at the cmip6deck experiments for clm5_0 (which is what cmip6 was run with). We don't think that is useful any more to maintain -- however starting to migrate these to cmip7 deck experiments DOES make sense. So we plan to label these as cmip7 with clm6_0, but leave them mostly as they are. As we find out what the cmip7 deck is going to look like -- we'll evolve them to look like the cmip7 deck. Let us know if you'd like to discuss this further if you aren't sure about that plan.
